src/handlers/index.ts

```diff
@@ -19,7 +22,12 @@ const functions = {
   getConfig: getConfigHandler,
   updateConfig: updateConfigHandler,
   getYnabAccountData,
+  electronGoogleLogin: googleLoginHandler,
```

Comment:
Having function names consistently as verbs makes the code more readable.
So I suggest. electronGoogleLogin -> loginToGoogle / initiateGoogleLogin etc